web/modules/custom/dpl_webmaster/dpl_webmaster.links.action.yml

```diff
@@ -0,0 +1,6 @@
+dpl_webmaster.install_or_update:
+  route_name: dpl_webmaster.install_or_update
+  title: 'Add new or update module'
+  weight: 25
```

Comment:
Consider adding a code comment explaining why this weight is necessary.

Weights appear as somewhat arbitrary numbers but they are often set to ensure that something occurs before or after another key point. Understanding this non-obvious relationship can be import in future work.

Reply:
You're right. Actually it's copied directly from update modules actions, but I can't see any obvious reason. I'll try to remove it.